org/mozilla/jss/crypto/RSAParameterSpec.java

```diff
@@ -4,12 +4,13 @@
 package org.mozilla.jss.crypto;
 
 import java.security.spec.AlgorithmParameterSpec;
+import java.security.spec.RSAKeyGenParameterSpec;
 import java.math.BigInteger;
 
 /**
  * This class specifies the parameters used for generating an RSA key pair.
  */
-public class RSAParameterSpec implements AlgorithmParameterSpec {
+public class RSAParameterSpec extends RSAKeyGenParameterSpec implements AlgorithmParameterSpec {
```

Comment:
Since [`RSAKeyGenParameterSpec`](https://docs.oracle.com/javase/8/docs/api/java/security/spec/RSAKeyGenParameterSpec.html) implements `AlgorithmParameterSpec`, we can drop the `implements AlgorithmParameterSpec` here.